doc/tutorial/network_ring.rst

```diff
@@ -0,0 +1,337 @@
+.. _tutorialnetworkring:
+
+A ring network
+==============
+
+In this example, a small *network* of cells, arranged in a ring, will be created and the simulation distributed over multiple threads or GPUs if available. Data on execution time and memory consumption will be recorded.
+
+.. Note::
+
+   **Concepts covered in this example:**
+
+   1. Building a basic :class:`arbor.cell` with a synapse site.
+   2. Building a :class:`arbor.recipe` with a network of interconnected cells.
+   3. Building an :class:`arbor.context` and a :class:`arbor.domain_decomposition`
+      that distributes the simulation over multiple threads or GPU.
+   4. Profile the simulation using a :class:`arbor.meter_manager`.
+   5. Create and execute a :class:`arbor.simulation`.
+   6. Running the simulation and visualizing the results,
+
+We outline the following steps of this example:
+
+1. Define the **cell**.
+2. Define the **recipe** of the model.
+3. Define the **execution context** of the model: a description of the underlying system
+   on which the simulation will run.
+4. Define the **meter manager** that will record execution time and memory consumption.
+5. Define the **domain decomposition** of the network: how the cells are distributed on
+   the different ranks of the system.
+6. Define the **simulation**.
+7. **Run** the simulation.
+8. Collect and visualize the **results**.
+
+The cell
```

Comment:
Could we simplify this by using a cell model developed in an earlier tutorial? We can change the existing ring example, or create a new one, and we don't need to make a fully stand alone page in the docs for each example.

Then you could say something like
```
The ring model that we will develop will have 4 cells based on the single cell model developed earlier <link>.
We wrap this in a function that returns the cell.
```
Then you could have a code sample that shows the function that generates the cell. That could later
be called in the recipe?

Reply:
I've thought of having a small library where for instance this and the Allen fit.json loader could be parked.

In this case, we're creating the first network in the tutorial section, so a simple cell seems a good choice.